cmd/otelcontribcol/builder-config.yaml

```diff
@@ -278,4 +278,6 @@ providers:
 # When using `make genotelcontribcol`, the `replaces` content is appended to this
```

Comment:
Please revert this change to keep git history clean, you can do that in a separate PR if you like.